src/libratbag-data.c

```diff
@@ -83,8 +89,10 @@ struct ratbag_device_data {
 	int refcount;
 	char *name;
 	char *driver;
+	char *type;
```

Comment:
maybe `*device_type` to make it a little bit more clear :)

Reply:
should I keep the enum as `devicetype` and the string as `device_type`? been thinking about this for a while

Reply:
I think that should be fine, even if it's not super beatiful